pkg/logging/mtlogging.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package logging
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+	"knative.dev/pkg/logging"
+)
+
+type MTLogger struct {
+	*zap.SugaredLogger                    // Logger for system
+	tenantLogger       *zap.SugaredLogger //Logger providing the tenant key
+	clientID           string             //client id usually the namespace
+}
+
+const tenantLoggingKey = "tenantNamespace"
+
+func (m MTLogger) TenantInfof(template string, args ...interface{}) {
```

Comment:
why are you adding the `Tenant` prefix? Just use the Logger interface like zap.SugaredLogger implements?